EMacco/abarian-backend
# #170468623 Update address

#### What does this PR do?
This PR allows a user to be able to update his address

#### Description of Task to be completed?
- make sure user is logged in
- make sure user is verified
- make sure that the description is present
- update the address

#### How should this be manually tested?
- checkout to this branch 
- send a patch request to `/addresses/:addressId` including the updated address
- the address should be updated and returned

#### Any background context you want to provide?
- you should be logged in
- you should be verified
- you should have an existing address

#### What are the relevant pivotal tracker stories?
[#170468623](https://www.pivotaltracker.com/story/show/170468623)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
